runtime/doc/eval.txt

```diff
@@ -1386,6 +1386,19 @@ v:errors	Errors found by assert functions, such as |assert_true()|.
 <		If v:errors is set to anything but a list it is made an empty
 		list by the assert function.
 
+					*v:event*
+v:event		Data related to an |autocommand| event as a Dictionary. The
+		available keys differ between different events and are
+		specificed at the documentation for each |event|. The possible
+		keys are:
+	regname 	requested register ["x] or the empty string for an
```

Comment:
Indentation does not look right, check `v:fcs_reason`. Also use proper sentences *always* or *never* in a list, this should be “Requested register (e.g. "x" for "xyy) or the empty string for an unnamed operation.” because you probably cannot shrink `operator` description.